contracts/extensions/GovernorSequentialProposalIdUpgradeable.sol

```diff
@@ -30,8 +31,7 @@ abstract contract GovernorSequentialProposalIdUpgradeable is GovernorUpgradeable
         mapping(uint256 proposalId => ProposalDetails) _proposalDetails;
     }
 
-    // keccak256(abi.encode(uint256(keccak256("openzeppelin.storage.GovernorSequentialProposalIdStorage")) - 1)) &
-    // ~bytes32(uint256(0xff))
+    // keccak256(abi.encode(uint256(keccak256("storage.GovernorSequentialProposalIdStorage")) - 1)) & ~bytes32(uint256(0xff))
```

Comment:
@jferas i'm concerned that the string value changed here but the hash below it didn't. Which one is incorrect? we should make sure the location hash matches the keccak comment.

Reply:
Thanks.. good catch in terms of minimizing change in the PR! That extra bit was because I was just shortening the line to avoid wrapping.. but it was misleading with regard to matching the string value in the erc7201 custom storage location directive.

Done: [8e94a4b](https://github.com/ScopeLift/compound-governance-upgrade/pull/82/commits/8e94a4b29c8a21052d8bbd4f559e6e4cfc15ce5a)

Reply:
He means recomputing the `GovernorSequentialProposalIdStorageLocation` hash value below according to the updated natspec. Which should be `0x357e1d0c89980520b3654c57f444238d75a15e5f41d389a090caabe54617d800`.

Reply:
Ah.. understood.. done: [75e801a](https://github.com/ScopeLift/compound-governance-upgrade/pull/82/commits/75e801ad90dd84a8a43535e920793f8b3d5d83bc)